client/components/site-stats-sticky-link/index.jsx

```diff
@@ -36,11 +39,23 @@ var SiteStatsStickyLink = React.createClass( {
 		}
 	},
 
+	preload: function() {
```

Comment:
It could be nice to abstract this out somehow, possibly using a higher order function/component?